src/classes/player.py

```diff
@@ -0,0 +1,62 @@
+"""Module that implements Player class."""
+
+
+class Player:
+    """Class that implements player."""
+
+    id = int
+    name = str
+    is_adm = bool
+    games = int
+    pitch = float
```

Comment:
I believe this is a bug, instead of annotations you get objects which are classes.

Reply:
Done